cbc.rb

```diff
@@ -0,0 +1,54 @@
+class Cbc < Formula
+  desc "Linear programming solver"
```

Comment:
This is still the old `desc`. I suggest we keep it to one formula per pull request when possible.